internal/display/tenants.go

```diff
@@ -28,8 +30,9 @@ func (r *Renderer) TenantList(data []string) {
 	var results []View
 	for _, item := range data {
 		results = append(results, &tenantView{
-			Name: item,
-			raw:  item,
+			Active: item == r.Tenant,
```

Comment:
Maybe prevent red X from appearing here?

Reply:
Good idea!

Reply:
I also credit @m3talsmith for the suggestion!

Modified the code to remove the red X. Hope it makes the active status more clear.